doc/source/whats_new.rst

```diff
@@ -31,6 +31,9 @@ Changelog
 
     - Add new object :class:`mne.decoding.TimeDecoding` for decoding sensors' evoked response across time by `Jean-Remi King`_
 
+    - Add support to append new channels from a list of Raw objects by `Chris Holdgraf`_
```

Comment:
And I think that from now on 'buffer_size_sec' should always be in any new info object, since it is now in RAW_INFO_FIELDS, right?